js/defaultAzureCredentialAdapter.spec.ts

```diff
@@ -0,0 +1,25 @@
+import assert from "assert";
+import * as dotenv from "dotenv";
+import { KeyVaultManagementClient } from "@azure/arm-keyvault";
+import { DefaultAzureCredentialAdapter } from "./defaultAzureCredentialAdapter";
+
+dotenv.config();
+
+const subscriptionId = process.env.AZURE_SUBSCRIPTION_ID;
+
+describe("DefaultAzureCredentialAdapter", function() {
```

Comment:
Let's not introduce key vault into the tests.  Please have a test that has the same logic as this test: https://github.com/jongio/azidext/blob/master/net/JonGallant.Azure.Identity.Extensions.Tests/Mgmt/ResourceGroupTests.cs